package.json

```diff
@@ -447,9 +460,9 @@
                 "default": null
               },
               "runspaceId": {
-                "type": "number",
+                "type": "string",
```

Comment:
I really wish it were possible to do like `string | int` for this.

Reply:
actually... it looks like you can do:

```
"type":["string","number"]
```

Reply:
@adamdriscoll I think you maybe missed this?

Reply:
Yep. In there now.